content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Virtual Machine Installation
-description: Deploy Istio and connect a workload running within a virtual machine to it.
+title: Virtual Machine Installation [Experimental]
+description: Deploy Istio and connect a workload running within a virtual machine to it (experimental).
```

Comment:
Line 17 says this feature is `alpha`, not experimental.

Reply:
Fixed